### DISCUSSION TITLE
Allow lanterns to attach to rope

### DISCUSSION THREAD

**Discussion by wjrogers** [vectorwing/FarmersDelight#729] [Ideas]
I would like to use ropes as a way to hang lanterns from the ceiling! Currently, trying to place a lantern on the bottom of a rope has no effect.

**Comment by vectorwing** [comment]
Yeah, that's a limitation with lanterns; they require the support to have a solid base, which requires the rope to not be pass-through to players. :(

Supplementaries adds a mixin for that, and I once discussed with the author about nabbing that code snippet for FD. I'll try to remember it for the backlog!

**Comment by vectorwing** [comment]
@wjrogers, 1.2.2 has recently been released with this feature. The author of Supplementaries helped me figure out how I could get Rope to support blocks beneath itself. Enjoy! 👍

**Comment by wjrogers** [comment]
Thank you!
